kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -191,6 +191,12 @@ public void checkpointed(long l)
   }
 
   @Override
+  public void beforeCheckpoint(long windowId)
```

Comment:
JIRA represents to update the checkpointlistner. In this operator, there is no functionality in checkpoint.